README.md

```diff
@@ -409,6 +419,22 @@ exports.handler = (event, context, callback) => {
 };
 ```
 
+## HTTP Exception handling
```

Comment:
I would like to ask @dchai76 to take a look @ this explanation too. For me, it's fine.

Reply:
Let's capitalizing `Handling` also